internal/cmd/test.go

```diff
@@ -135,3 +170,28 @@ func runCMD(mainFunc func(), args []string) (err error) {
 
 	return err
 }
+
+func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, commonCases CommonCMDTestCases, prefix string) (string, string, int, bool) {
+	stdout, stderr, exitCode, err := RunCMDTest(test, mainFunc, args)
+	if err != nil {
+		test.Errorf("%sCMD run returned an error: '%v'.", prefix, err)
+		return "", "", -1, false
+	}
+
+	if !strings.Contains(stderr, commonCases.ExpectedStderr) {
```

Comment:
!!!
We explicitly talked about this!
This name is bad.

As a reviewer, the fact that you missed this is very concerning
(since we spent so much time talking about it).
Do what you need to do (take notes, record conversations, ask questions, etc) to never make a mistake like this again.